lite-locals.tf

```diff
@@ -0,0 +1,470 @@
+      minDeltaSize = 50000
+      nodeSelector = var.aks_4nodepools? { workload = "servpool" } : { workload = "hpccpool" }
+      resources = {
+        cpu    = "1"
+        memory = "4G"
+      }
+    }
+    resources = {
+      cpu    = "2"
+      memory = "8G"
+    }
+    maxStartupTime = 1200
+  }
+
+  dfuserver_settings = {
+    maxJobs = 3
+    nodeSelector = var.aks_4nodepools? { workload = "servpool" } : { workload = "hpccpool" }
+    resources = {
+      cpu    = "1"
+      memory = "2G"
+    }
+  }
+
+  sasha_config = {
+    disabled = false
+    nodeSelector = var.aks_4nodepools? { workload = "servpool" } : { workload = "hpccpool" }
+    wu-archiver = {
+      disabled = false
+      service = {
+        servicePort = 8877
+      }
+      plane           = "sasha"
+      interval        = 6
+      limit           = 400
+      cutoff          = 3
+      backup          = 0
+      at              = "* * * * *"
+      throttle        = 0
+      retryinterval   = 6
+      keepResultFiles = false
+    }
+
+    dfuwu-archiver = {
+      disabled = false
+      service = {
+        servicePort = 8877
+      }
+      plane    = "sasha"
+      interval = 24
+      limit    = 100
+      cutoff   = 14
+      at       = "* * * * *"
+      throttle = 0
+    }
+
+    dfurecovery-archiver = {
+      disabled = false
+      interval = 12
+      limit    = 20
+      cutoff   = 4
+      at       = "* * * * *"
+    }
+
+    file-expiry = {
+      disabled             = false
+      interval             = 1
+      at                   = "* * * * *"
+      persistExpiryDefault = 7
+      expiryDefault        = 4
+      user                 = "sasha"
+    }
+  }
+
+  ldap_config = null
+
+  ldap_tunables = {
+    cacheTimeout                  = 5
+    checkScopeScans               = false
+    ldapTimeoutSecs               = 131
+    maxConnections                = 10
+    passwordExpirationWarningDays = 10
+    sharedCache                   = true
+  }
+
+  install_blob_csi_driver = true
+
+  remote_storage_plane = null
+
+  onprem_lz_settings = {}
+
+  admin_services_node_selector = {}
+
+  thor_config = [{
+    disabled            = (var.enable_thor == true) || (var.enable_thor == null)? false : true
+    name                = "thor"
+    prefix              = "thor"
+    numWorkers          = var.thor_num_workers
+    keepJobs            = "none"
+    maxJobs             = var.thor_max_jobs
+    maxGraphs           = 2
+    maxGraphStartupTime = 172800
+    numWorkersPerPod    = 1
+    nodeSelector        = var.aks_4nodepools? { workload = "thorpool" } : { workload = "hpccpool" }
+    egress              = "engineEgress"
+    tolerations_value   = "thorpool"
+    managerResources = {
+      cpu    = 1
+      memory = "2G"
+    }
+    workerResources = {
+      cpu    = local.thor_worker_cpus
+      memory = format("%dG", local.thor_worker_ram)
+    }
+    workerMemory = {
+      query      = "3G"
+      thirdParty = "500M"
```

Comment:
The query value is really workerResources.memory - thirdParty, rounded down.  Please change the hardcoded 3G to that computation.

The goal of this block is to divide workResources.memory into "memory for Thor" and "memory for 3rd party tools".

Reply:
In your commit 81eca56d96befcdf4148fbb58a13f91ea19e429a you changed the query value to exactly the same value as workerResources.memory, which is not what was requested.  Your change runs the risk of over-allocation of RAM, leading to an Out Of Memory exception and a killed pod.